plugins/vpp/l3plugin/vppdump/dump_vppcalls.go

```diff
@@ -39,7 +39,7 @@ func DumpStaticRoutes(log logging.Logger, vppChan govppapi.Channel, timeLog meas
 		}
 	}()
 
-	var routes []*vppcalls.Route
+	var routes [][]*vppcalls.Route
```

Comment:
this solution seems not very readable for me, also you will have a lot of duplicate fields in the resulting list. 
The destination address, table ID (Vrf) and table name are the same for every path. Path fields are next-hop, interface, weight and preference. I'd prefer to create a new struct Path (for example) with all these fields, and add a list of Path objects to Route.

Reply:
I agree. Having simple struct even with single slice field will be much more readable because it can be named and documented more easily.